### PR TITLE
MOSIP-32461, MOSIP-36412 Changing commons  versions for 1.3.x branch

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: commons-packet/commons-packet-service/pom.xml [hunk 1]
```diff
@@ -30,14 +30,14 @@
         <kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>
```
Comment (ashok-ksharma) [inline commons-packet/commons-packet-service/pom.xml:30]: Are we using this:
<kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>

File: commons-packet/commons-packet-service/pom.xml [hunk 2]
```diff
@@ -30,14 +30,14 @@
...
         <swagger.version>2.9.2</swagger.version>
 
-        <kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
+        <kernel.core.version>1.3.0-beta.1</kernel.core.version>
         <kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>
         <kernel.crypto-jce.version>1.3.0-SNAPSHOT</kernel.crypto-jce.version>
```
Comment (ashok-ksharma) [inline commons-packet/commons-packet-service/pom.xml:35]: Are we using below:
 <kernel.crypto-jce.version>1.3.0-SNAPSHOT</kernel.crypto-jce.version>

File: commons-packet/commons-packet-service/pom.xml [hunk 3]
```diff
@@ -30,14 +30,14 @@
...
         <kernel.auth.adaptor.version>1.3.0-SNAPSHOT</kernel.auth.adaptor.version>
-        <kernel-dataaccess-hibernate.version>1.3.0-SNAPSHOT</kernel-dataaccess-hibernate.version>
+        <kernel-dataaccess-hibernate.version>1.3.0-beta.1</kernel-dataaccess-hibernate.version>
         <kernel.logger.logback.version>1.3.0-SNAPSHOT</kernel.logger.logback.version>
```
Comment (ashok-ksharma) [inline commons-packet/commons-packet-service/pom.xml:38]: Change <kernel.logger.logback.version> to 1.3.0-beta.1

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: commons-packet/commons-packet-service/pom.xml (hunk 3 above)
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-service/pom.xml:38]: done

File: commons-packet/commons-packet-service/pom.xml (hunk 2 above)
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-service/pom.xml:35]: not using this property. hence removing it.

File: commons-packet/commons-packet-manager/pom.xml [hunk 4]
```diff
@@ -29,17 +29,17 @@
...
 
-        <kernel-keymanager-service.version>1.3.0-SNAPSHOT</kernel-keymanager-service.version>
-        <kernel-idobjectvalidator.version>1.3.0-SNAPSHOT</kernel-idobjectvalidator.version>
-        <kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
+        <kernel-keymanager-service.version>1.3.0-beta.1</kernel-keymanager-service.version>
+        <kernel-idobjectvalidator.version>1.3.0-beta.1</kernel-idobjectvalidator.version>
+        <kernel.core.version>1.3.0-beta.1</kernel.core.version>
         <kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>
         <kernel.cbeffutil.api.version>1.3.0-SNAPSHOT</kernel.cbeffutil.api.version>
         <kernel.crypto-jce.version>1.3.0-SNAPSHOT</kernel.crypto-jce.version>
```
[the remark this answers is not in the document]
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-manager/pom.xml:37]: not using this property. hence removing it.

File: commons-packet/commons-packet-manager/pom.xml [hunk 5]
```diff
@@ -29,17 +29,17 @@
         <kernel.keygenerator.bouncycastle.version>1.3.0-SNAPSHOT</kernel.keygenerator.bouncycastle.version>
```
[the remark this answers is not in the document]
Comment (Khuddusshariff0022) [reply commons-packet/commons-packet-manager/pom.xml:29]: not using this property. hence removing it.

**Review by ashok-ksharma — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**